react/SearchContext.js

```diff
@@ -118,6 +118,7 @@ const SearchContext = ({
           to,
           // backwards-compatibility
           rest,
+          maxItemsPerPage,
```

Comment:
place this before the `// backwards-compatibility`

Reply:
:rage1: